src/applications/proxy-rewrite/sass/_m-vet-nav-px.scss

```diff
@@ -97,23 +97,6 @@ body.va-pos-fixed {
     margin-bottom: 0;
   }
 
-  .va-button-primary {
-    border: 2px solid $color-white;
```

Comment:
I checked the injected header locally and inspected the `#vetnav-menu` wrapping element. I did not see anywhere that this class was still being used so I think it's safe to remove it from here now:

![Screenshot 2024-01-09 at 9 49 32 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/71c9a682-f5b9-4303-86d3-84554e053f2e)